test/instructions/duphash_test.rb

```diff
@@ -4,8 +4,26 @@
 
 class TenderJIT
   class DuphashTest < JITTest
+    def duphash
+      a = {a: 1, b: 2}
```

Comment:
I think `a =` can be omitted, as it doesn't affect the test, and it yields a simpler bytecode.

On 3.0.2, the two variants disassemble to:

```
3.0.2 :001 > puts RubyVM::InstructionSequence.compile(%q[ a = {a: 1, b: 2}   ]).disasm
== disasm: #<ISeq:<compiled>@<compiled>:1 (1,0)-(1,17)> (catch: FALSE)
local table (size: 1, argc: 0 [opts: 0, rest: -1, post: 0, block: -1, kw: -1@-1, kwrest: -1])
[ 1] a@0
0000 duphash                                {:a=>1, :b=>2}            (   1)[Li]
0002 dup
0003 setlocal_WC_0                          a@0
0005 leave

3.0.2 :002 > puts RubyVM::InstructionSequence.compile(%q[ {a: 1, b: 2}   ]).disasm
== disasm: #<ISeq:<compiled>@<compiled>:1 (1,0)-(1,13)> (catch: FALSE)
0000 duphash                                {:a=>1, :b=>2}            (   1)[Li]
0002 leave
```